packages/account-v2/src/index.scss

```diff
@@ -2,7 +2,6 @@
 @tailwind components;
 @tailwind utilities;
 
-@import '@deriv/quill-design/dist/quill-design.css';
```

Comment:
`quill-design.css` bundles up default tailwind stylings, thus it causes some of the root stylings in core package to be overwritten. For instance, the Deposit button here has a transparent background due to tailwind's overriding the styling:

https://github.com/binary-com/deriv-app/assets/103016120/0644b066-1f91-4cfe-a70c-e26d6254f6d2